### PR TITLE
Increase Prometheus coverage

### PR DESCRIPTION
**Pull request by riemass**
It seems we only use Prometheus in `robot/middleman/driver.cpp`. 

The reason we have so low coverage for Prometheus is that the `robot` folder is in the ignore list. Bad news is that removing the ignore on this folder this will also expect the drivers to be covered, so it's best to let the pipeline pass once and see the result.

### CODE REVIEW

**Comment by Neverlord** [comment]
Even if the `robot` folder itself is ignored, the coverage from the other folders will still be counted. So this change will only add noise.

Are you maybe confusing the `caf.io` and the `caf.net` Prometheus implementation? The test you mention will cover the Prometheus Broker and has nothing to do with the implementation in `caf.net` https://app.codecov.io/gh/actor-framework/actor-framework/blob/master/libcaf_io%2Fcaf%2Fdetail%2Fprometheus_broker.cpp.

We simply don't have tests yet that execute the `caf.net` Prometheus code.

**Comment by riemass** [comment]
You're right, I got the `net` and `io` drivers mixed up. I've enabled Prometheus in our `caf::net` robot tests to see the result.

**Comment by riemass** [comment]
The `codecov` shows 100% on `prometheus.hpp/cpp` now. @Neverlord can I leave the test in `rest.robot`, or should I factor it out to a new file?

File: robot/http/rest.robot [hunk 1]
```diff
@@ -14,6 +14,11 @@ ${BINARY_PATH}          /path/to/the/server
 ${SSL_PATH}             /path/to/the/pem/files
 ${MAX_REQUEST_SIZE}     2048
 
+# Prometheus variables
+${PR_SERVER_PORT}       55515
+${PR_HTTP_URL}          http://localhost:55515/metrics
+${PR_HTTP_BAD_URL}      http://localhost:55515/foobar
```
Comment (Neverlord) [inline robot/http/rest.robot:20]: Please use port 55516. This will collide when running test suites in parallel.

**Review by Neverlord — CHANGES_REQUESTED** [review]

**Comment by Neverlord** [comment]
> can I leave the test in `rest.robot`, or should I factor it out to a new file?

Can stay, just use a different port. Also, please squash the commits since the changes of the first commit have been reverted anyways.

**Comment by riemass** [comment]
Rebased and squashed.

**Review by Neverlord — APPROVED** [review]

**Pull request merged**